aiounifi/interfaces/api_handlers.py

```diff
@@ -98,8 +98,15 @@ def __init__(self, controller: "Controller") -> None:
     @final
     async def update(self) -> None:
         """Refresh data."""
-        raw = await self.controller.request(self.api_request)
-        self.process_raw(raw.get("data", []))
+        data = await self.controller.request(self.api_request)
+        self.process_typed_api_response(data)
```

Comment:
Is this change really necessary, I don't understand why this change is needed. From what I can tell you do exactly the same thing, process_raw only iterates over an iterator and then passes each item along to process_item

Reply:
I think this has to do with me messing the merges up. This is indeed not needed.